src-cpp/server42/Dockerfile

```diff
@@ -12,4 +12,6 @@ WORKDIR /root/cmake-build-release/
 
 COPY ./docker_entrypoint.sh /root/cmake-build-release/
```

Comment:
should be removed since `.sh` file does not exist anymore. This fails the image build atm

Reply:
done